tasks/disk_drive.yml

```diff
@@ -1,6 +1,9 @@
 ---
 - name: assert that we only run on HP
-  assert: { that: "ansible_system_vendor == 'HP'" }
+  assert: { that: "ansible_system_vendor == 'HP' or 'HPE'" }
+
+- name: Choose repository for the firmware
+  set_fact: hp_repository={{ item.hp_repository }}
```

Comment:
`item` is not defined when running disk_drive.yml, should these two lines be removed?

Reply:
No, you are supposed to define it in group_vars or host_vars if you wish to update firmware. Look at the example in Readme.md.

Reply:
Yes, you are right.